.github/ISSUE_TEMPLATE/release.md

```diff
@@ -69,14 +69,20 @@ production readiness.
   > `v0.5.10-dev` -> `v0.6.0` when they upgrade than upgrading and going from `v0.6.0-dev` ->
   > `v0.5.10`.
 
-  - [ ] Open a PR with this change, and land it.
+  - [ ] Related to the above note, **if this is a feature release** you must
```

Comment:
"Feature release" is not defined anywhere in the document.

Reply:
good point, added a brief phrase mentioning that it's a feature release if product says it is.